src/lib.rs

```diff
@@ -470,9 +470,12 @@ pub trait Labeller<'a,N,E> {
     }
 
     /// Maps `e` to arrow style that will be used on the end of an edge.
-    /// Defaults to normal.
+    /// Defaults to normal or none, depending on `kind()`.
     fn edge_end_arrow(&'a self, _e: &E) -> Arrow {
```

Comment:
Perhaps we should create a new arrow type 'default', which doesn't create any `arrowhead=` or `arrowtail=` annotations.  That way we don't need to explicitly worry about the graph kind here.